README.md

```diff
@@ -383,15 +385,13 @@ For example in `.eslintrc.*`:
       "parser": "svelte-eslint-parser",
       "parserOptions": {
         "parser": "...",
-        "svelteFeatures": { "runes": true },
```

Comment:
I think 95% of users doesn't need to specify this option, so I removed from README.